apps/strapi-cms/config/database.ts

```diff
@@ -1,11 +1,41 @@
 import path from 'path';
 
-export default ({ env }: any) => ({
-  connection: {
-    client: 'sqlite',
+export default ({ env }: any) => {
+  const client: string = env('DATABASE_CLIENT');
+
+  const connections: {[key: string]: any} = {
+    sqlite: {
```

Comment:
I'm dropping here a comment just to let you know:
I've created [this activity](https://pagopa.atlassian.net/browse/B2BP-198) with the goal to remove `sqlite` block.

At the moment it's ok to have it, but since we are going to use `postgres`, it's better to have the very same environment locally as the one we are going to deploy.
So the activity created aims to do that.

But still, at the moment we can keep `sqlite` as well! 😄

Reply:
Sounds good, thank you!

I'll jump on that ASAP, updating the README as well.

Reply:
Don't worry, we can handle that in another sprint, because IMHO we should find an easy way to set up Postgres DB too (maybe with a Dockerfile? I don't know yet). My intent was just to let you know 😄